packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -153,8 +153,8 @@ export async function processOutboundDocumentQueryResps({
 
     await setDocQueryProgress({
       patient: { id: patientId, cxId: cxId },
-      downloadProgress: { status: "failed" },
-      convertProgress: { status: "failed" },
+      downloadProgress: { status: "failed", total: 0 },
```

Comment:
Updated this but not only the total is set to 0 but also the succ and errors because if those have values and total is 0 we will also get the warning on cleanup